src/app/hooks/useOptimizelyVariation/useServerSide/index.test.tsx

```diff
@@ -13,41 +18,55 @@ const optimizely = {
   setUser: jest.fn(() => Promise.resolve()),
 };
 
-describe('useOptimizelyMvtVariation custom hook', () => {
+describe('useOptimizely - useServerSide', () => {
   beforeEach(() => {
     jest.clearAllMocks();
   });
 
-  const renderUseOptimizelyMvtVariation = (mvtExperiments, flagId) => {
+  const renderUseServerSide = (params: {
+    experimentName: string;
+    serverSideExperiments?: ServerSideExperiment[];
+  }) => {
+    const { experimentName, serverSideExperiments } = params;
+
     const props = {
-      mvtExperiments,
+      serverSideExperiments,
       isAmp: false,
-      pageType: 'STY',
-      service: 'foo',
+      pageType: 'STY' as PageTypes,
+      service: 'news' as Services,
       pathname: 'bar',
     };
-    const wrapper = ({ children }) => (
-      <OptimizelyProvider optimizely={optimizely} isServerSide>
+    const wrapper = ({ children }: PropsWithChildren) => (
+      <OptimizelyProvider
+        optimizely={optimizely as unknown as ReactSDKClient}
+        isServerSide
+      >
         <RequestContextProvider {...props}>{children}</RequestContextProvider>
       </OptimizelyProvider>
     );
-    return renderHook(() => useOptimizelyMvtVariation(flagId), {
+    return renderHook(() => useServerSide(experimentName), {
```

Comment:
As above - should we be using `renderHook` from `'@testing-library/react-hooks/server'` -  https://github.com/bbc/simorgh/blob/6ab6f3db1add8163fb91a70de3378ac3fcd0a3c9/src/app/hooks/useHydrationDetection/index.test.tsx#L5 - a server hook renderer instead?